packages/subgraph/src/v1-analytics/schema.graphql

```diff
@@ -90,6 +92,12 @@ type HourlyMetric @entity {
   liquidity: BigInt!
   # hourly number of transactions per asset
```

Comment:
if my impression is correct, the more data we store, the more costly the subgraph... so couldn't this value, `txCount` be calculated as the sum of both the `sending` and the `receiving` ?

Reply:
the more costly in terms of time per request (eventually perhaps for the actual query when it becomes a fully decentralized network, but we're a bit far off from that). i don't think the latency here matters so much, but you could count it as the sending and receiving. i prefer the convenience/clarity here though

Reply:
gotcha kk nw